### ISSUE TITLE
\x1b[?1001r sequence resets cursor to the beginning 

### ISSUE THREAD

**Issue by thestr4ng3r** [asciinema/asciinema-player#122]
Reproducer: [mouse.sh.gz](https://github.com/asciinema/asciinema-player/files/3869727/mouse.sh.gz)
Output should look like this:
```
line a----------
line b----------
line c----------
line d 123
line e
```

Example Recording: https://asciinema.org/a/HlH7tRw4wAqGeFvbl7z6Mx2mN
When playing the .cast file in the terminal, it looks fine.

Unfortunately, this currently breaks all recordings created with [radare2](https://github.com/radareorg/radare2)

**Comment by sickill** [comment]
This has been fixed in `develop`, will be released in v3.0.0-beta.3 very soon.

**Comment by thestr4ng3r** [comment]
Thanks!
